priorityq/priorityq_test.go

```diff
@@ -0,0 +1 @@
+package priorityq
```

Comment:
Tests have to check whether the priority queue works in the right way. So, every method should be checked un separate test function. Note that you shouldn't check them as `pq.Pop()` or something like that, use `heap.Pop(&pq)`.